src/EA.Weee.Web/Areas/AatfReturn/Controllers/ReturnsController.cs

```diff
@@ -66,5 +66,11 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
             breadcrumb.ExternalActivity = activity;
             breadcrumb.SchemeInfo = await cache.FetchSchemePublicInfo(organisationId);
         }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Continue(Guid organisationId, Guid returnId, bool redirectReportingOptions, bool redirectSelectYourPcs, bool redirectTaskLis)
+        {
+            return AatfRedirect.SelectReportOptions(organisationId, returnId);
```

Comment:
Unit test to make sure the user is redirected?

Reply:
Changed the way this works.